web/static/js/reducers/users.js

```diff
@@ -39,3 +39,8 @@ const users = (state = [], action) => {
 }
 
 export default users
+
+// SELECTORS
+export const getUser = (state, userId) => (
+  state.users ? state.users.find(user => user.id === userId) : {}
```

Comment:
is there a case where we wouldn't have users?

Reply:
the selector was introduced by Damon. I only changed a part of it that was breaking tests (it used to return 'null' in the 'else' portion of the ternary but I changed it to '{}'.

Reply:
weird. i don't see an associated test for this selector. it's probably due to us testing the `connect`ed component (which will trigger the `mapStateToProps` call that uses this selector. Rather than change the implementation to accommodate an unrelated test, you could export the unconnected component and test that.

Reply:
We added a test for this function. Can you elaborate on what you meant by exporting the unconnected component to test that?

Reply:
for sure, but please note that I was only guessing at the cause of the issue Tanya mentioned above, so it could be wrong. Regarding exporting unconnected components, I think this doc sums it up better than I can: https://github.com/reactjs/redux/blob/master/docs/recipes/WritingTests.md#connected-components

For an example in the codebase, peep [Alert.jsx](https://github.com/stride-nyc/remote_retro/blob/master/web/static/js/components/alert.jsx) and [its associated test](https://github.com/stride-nyc/remote_retro/blob/master/test/components/alert_test.js)